### PR TITLE
added data and map

### PR DESCRIPTION
**Pull request by suademre**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,37 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Header from "./components/header/Header";
+import Main from "./components/main/Main";
+import Footer from "./components/footer/Footer";
+import data from "./data.json";
 
 function App() {
+  console.log(data);
```
Comment (macfrei) [inline src/App.js]: Please don't forget to remove `console.log`s before merging.

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
+        house="Gryffindor"
+        imgUrl="http://hp-api.herokuapp.com/images/harry.jpg"
+        actor="Daniel Radcliffe"
+      />
+      <Main
+        characterName="Hermione Granger"
+        house="Gryffindor"
+        imgUrl="http://hp-api.herokuapp.com/images/hermione.jpeg"
+        actor="Emma Watson"
+      /> */}
```
Comment (macfrei) [inline src/App.js]: Since you don't need this code anymore, please remove it.

File: src/App.js [hunk 3]
```diff
@@ -1,23 +1,37 @@
...
+      {data.map((character) => (
+        <Main
```
Comment (macfrei) [inline src/App.js]: `Main` is an unusual name for a small component like this card. I think `Card` would be a better name. For example, I as a different developer looking at your code, would think that with this `Main` component you actually mean a component which includes other components as in the `main` HTML element.

File: src/components/footer/Footer.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import "./Footer.css";
+
+function Footer() {
+  return (
+    <footer className="footer">
+      <button>
```
Comment (macfrei) [inline src/components/footer/Footer.js]: Please add an `aria-label` to the `button`s as this is not accessible.

File: src/components/header/Header.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
+@import url("https://fonts.googleapis.com/css2?family=Nothing+You+Could+Do&display=swap");
+
+.header {
+  /* width: 100%;
+    position: fixed;
+    padding: 10px 20px;
+    justify-content: center;
+    background-color: #fff;
+    align-items: center;
+    box-shadow: 0px 0px 15px #dedede; */
```
Comment (macfrei) [inline src/components/header/Header.css]: Again, please remove unused and commented out code!

File: src/components/header/Header.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import "./Header.css";
+
+function Header() {
+    return (
+        <header className="header">
+            <h1>Herry Potter</h1>
```
Comment (macfrei) [inline src/components/header/Header.js:6]:
```suggestion
            <h1>Harry Potter</h1>
```

File: src/components/main/Main.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
+import "./Main.css";
+
+function tellColorName(house) {
+  /* house.forEach(house => {
+        
+    }) */
```
Comment (macfrei) [inline src/components/main/Main.js]: please remove!

File: src/components/main/Main.js [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+function Main({ characterName, house, imgUrl, actor }) {
+  const color = tellColorName(house);
+  console.log("color : " + color);
+  /* const herryName = "Herry Poter";
+  const herryActor = "Daniel Radcliffe";
+  const harryImgae = "http://hp-api.herokuapp.com/images/harry.jpg";
+
+  const hermioneName = "Hermione Granger";
+  const hermioneActor = "Emma Watson";
+  const hermioneImg = "http://hp-api.herokuapp.com/images/hermione.jpeg"; */
```
Comment (macfrei) [inline src/components/main/Main.js]: This should be removed!

File: src/components/main/Main.js [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+  return (
+    <>
+      <main>
+        {/* {data.map((d, key) => ( */}
```
Comment (macfrei) [inline src/components/main/Main.js]: Please remove.

File: src/components/main/Main.js [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+          </div>
+          <div className="card_propertiesTag">
+            <h4 className="card__name">Name : {characterName} </h4>
+            <h6 className="card__actor">Actor's Name{actor}</h6>
+            <h6 className={color}>House : {house}</h6>
+            <button className="more">More</button>
+          </div>
+        </section>
+        {/* ))} */}
+      </main>
```
Comment (macfrei) [inline src/components/main/Main.js]: One `Card` should not be in a `main` element. Remember, you are creating one component for each element in the array. So at the moment you have waaaaay more than one `main` element on your page. Which is _not_ ideal.

File: src/components/main/Main.js [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+            <img className="card__img__tag" src={imgUrl} alt={characterName} />
+          </div>
+          <div className="card_propertiesTag">
+            <h4 className="card__name">Name : {characterName} </h4>
+            <h6 className="card__actor">Actor's Name{actor}</h6>
+            <h6 className={color}>House : {house}</h6>
+            <button className="more">More</button>
+          </div>
+        </section>
+        {/* ))} */}
```
Comment (macfrei) [inline src/components/main/Main.js]: Please remove..

**Review by macfrei — CHANGES_REQUESTED** [review]
Hi Suad, please remember to delete code that you don't need and use anymore! You have a lot of commented out code and `console.log`s which shouldn't be merged into the `main` branch. I also found an issue regarding your `Main` component. Please consider my remarks and make some changes. 🙂 👍

File: src/App.js (hunk 1 above)
Comment (suademre) [reply src/App.js]: ich habe entfernt.

File: src/App.js (hunk 2 above)
Comment (suademre) [reply src/App.js]: ich habe entfernt.

File: src/App.js (hunk 3 above)
Comment (suademre) [reply src/App.js]: Ich habe geändert

File: src/components/footer/Footer.js (hunk 4 above)
Comment (suademre) [reply src/components/footer/Footer.js]: Ich habe hinzufügen

File: src/components/header/Header.css (hunk 5 above)
Comment (suademre) [reply src/components/header/Header.css]: ich habe entfernt

File: src/components/main/Main.js (hunk 7 above)
Comment (suademre) [reply src/components/main/Main.js]: Ich habe entfernt

File: src/components/main/Main.js (hunk 8 above)
Comment (suademre) [reply src/components/main/Main.js]: Ich habe entfernt

File: src/components/main/Main.js (hunk 9 above)
Comment (suademre) [reply src/components/main/Main.js]: Ich habe entfernt.

File: src/components/main/Main.js (hunk 10 above)
Comment (suademre) [reply src/components/main/Main.js]: Ich habe geändert

File: src/components/main/Main.js (hunk 11 above)
Comment (suademre) [reply src/components/main/Main.js]: Ich habe entfernt

**Pull request merged**